app/src/test/java/org/groundplatform/android/ui/datacollection/tasks/polygon/DrawAreaTaskFragmentTest.kt

```diff
@@ -85,6 +86,7 @@ class DrawAreaTaskFragmentTest :
     )
   }
 
+  @Ignore
```

Comment:
Did you mean to check the these in?

Reply:
planning to take this in next PR and take this pr in coming release.